Readme.md

```diff
@@ -2,15 +2,15 @@
 
 ![Hacktoberfest logo](docs/img/Hfest-Logo-2-Color-Manga.png)
 
-Hello all you beautiful nerds and welcome to another year of [Hacktoberfest](https://hacktoberfest.com/)! at the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com)
```

Comment:
I think that all of these proposed changes to the Readme file are of low value. I'd much rather appreciate you adding an organization to the Swag List from one of the Issues. Please un-do these propsed chages to the Readme you made; I will not be able to merge any _good contribution_ with this still in place.